apps/ledger-live-mobile/src/newArch/features/Accounts/components/AccountListView/components/AccountRow.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Flex, Tag, Text } from "@ledgerhq/native-ui";
+import ParentCurrencyIcon from "~/components/ParentCurrencyIcon";
+import BigNumber from "bignumber.js";
+import CounterValue from "~/components/CounterValue";
+import { Account, DerivationMode } from "@ledgerhq/types-live";
+import { useMaybeAccountName } from "~/reducers/wallet";
+import { getTagDerivationMode } from "@ledgerhq/coin-framework/lib/derivation";
+
+interface AccountRowProps {
+  account: Account;
+  balance: BigNumber;
+}
+
+const formatAddress = (address: string) => {
+  if (address.length > 11) {
+    return `${address.slice(0, 4)}...${address.slice(-4)}`;
+  }
+  return address;
+};
+
+const AccountRow: React.FC<AccountRowProps> = ({ account, balance }) => {
+  const currency = account.currency;
+  const accountName = useMaybeAccountName(account);
+  const formattedAddress = formatAddress(account.freshAddress);
+  const tag =
+    account.type === "Account" &&
+    account?.derivationMode !== undefined &&
+    account?.derivationMode !== null &&
+    currency.type === "CryptoCurrency" &&
+    getTagDerivationMode(currency, account.derivationMode as DerivationMode);
+
+  return (
+    <>
+      <Flex flex={1} flexShrink={1}>
+        <Flex flexDirection="row" columnGap={8} alignItems="center" maxWidth="70%">
+          <Text
```

Comment:
think about very long account name, expect an ellipsis based rendering here

Reply:
That's already the case 😄 
![image](https://github.com/user-attachments/assets/1eec6efa-0a75-43ce-9aca-fcfa36d68a1c)

Reply:
The three last are testnet that's why the width is different since there is no price